.wokeignore

```diff
@@ -1 +1,4 @@
-docs/
\ No newline at end of file
+docs/
+
+# wokeignore:rule=master
+templates/cdk.master.leader.file-watcher.*
```

Comment:
TIL of this `.wokeignore` syntax.  neat!

Reply:
this is actually the syntax to ignore a single baddy name... i'm actually woke-ignoring the `.wokeignore` file here.